src/Microsoft.Extensions.Caching.Memory/MemoryCache.cs

```diff
@@ -63,12 +66,8 @@ public int Count
             get { return _entries.Count; }
         }
 
-        public IEntryLink CreateLinkingScope()
-        {
-            return EntryLinkHelpers.CreateLinkingScope();
-        }
-
-        public object Set(object key, object value, MemoryCacheEntryOptions cacheEntryOptions)
+        /// <inheritdoc />
+        public void Remove(object key)
```

Comment:
A lot of the diff in this view is just caused by method re-ordering, correct? Can you revert that where possible? E.g. Remove, Dispose, etc.?